src/edge/pcr.py

```diff
@@ -164,16 +164,20 @@ def pcr_from_genome(genome, primer_a_sequence, primer_b_sequence):
                     uniq_products[k] = product
 
     if len(pcr_products) == 1:
-        product = pcr_products[0][0]
-        region = pcr_products[0][1]
+        product, template_info = pcr_products[0]
+        region = template_info["region"]
+        fragment = template_info["fragment"]  # already an indexed fragment
+        bp_lo, bp_hi = region
```

Comment:
we should extend the region we return by the length of the primers, otherwise we'd have nothing to bind to in the next round of PCR.

Reply:
Actually, can you verify thats what the outcome of
```
bs_start = (fwd_primer_res.subject_end + 1) - (
        fwd_primer_res.query_end - fwd_primer_res.query_start + 1
    )
    bs_end = (rev_primer_res.subject_end - 1) + (
        rev_primer_res.query_end - rev_primer_res.query_start + 1
    )
``` is?

Reply:
ok, i had some paper around, i think its good.

Reply:
👍